app/models/identity.rb

```diff
@@ -0,0 +1,13 @@
+class Identity < Base
```

Comment:
It may be clearer to have this match the step name, so `ReturningTeacher`

Reply:
Not sure about that, the Model names match the question they are asking, so the next step from Identity is 'are you returning to teaching' - ReturningTeacher

Reply:
Yep my bad - I misread and thought this step was 'returning teacher'